lib/documents/schemas/ukhsa_data_access_approvals.json

```
@@ -458,6 +458,10 @@
           "label": "HTLV Surveillance System",
           "value": "htlv-surveillance-system"
         },
+        {
+          "label": "Human Growth Hormone Recipients at Risk of iCJD",
+          "value": "human-growth-hormone-recipients-at-risk-of-iCJD"
```

Should probably be all lowercase

```suggestion
          "value": "human-growth-hormone-recipients-at-risk-of-icjd"
```